apache/docker-entrypoint.sh

```diff
@@ -71,6 +149,8 @@ if [ -n "$MAUTIC_CRON_DYNAMICS" ]; then
         echo "10,40 * * * *     www-data   php /var/www/html/app/console mautic:integration:fetchleads -i Dynamics > /var/log/cron.pipe 2>&1" >> /etc/cron.d/mautic
 fi
 
+echo >&2 "@reboot  www-data   [[ "$(ls -A /var/www/html/app/cache/ip_data 2>/dev/null)" ]] || php /var/www/html/app/console mautic:iplookup:download > /var/log/cron.pipe 2>&1"
```

Comment:
This line doesn't make sense to me. You need to add it to the cronfile or drop the cron syntax and execute it directly from bash.